src/installers/next-auth.ts

```diff
@@ -36,8 +36,12 @@ export const nextAuthInstaller: Installer = async (
     "src/pages/api/restricted.ts",
   );
 
+  const middlewareSrc = path.join(nextAuthAssetDir, "middleware.ts");
+  const middlewareDest = path.join(projectDir, "src/middleware.ts");
```

Comment:
Next middleware goes in `_middleware.ts`

Reply:
They changed it to middleware.ts on the latest stable release. And now it goes outside pages

Reply:
You get errors when trying to import: ![CleanShot 2022-07-06 at 01 26 23](https://user-images.githubusercontent.com/51714798/177433106-488fe5bc-2771-43d7-bfe3-0f6ea1d0ec52.png)